### PR TITLE
DOC-5507: Fixed broken links

### PR DESCRIPTION
**Pull request by amiegrace12**

### CODE REVIEW

File: content/en/platform/corda/5.0/application-networks/managing/registration-requests/configuring-manual-approval-rules.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ curl -k -u $REST_API_USER:$REST_API_PASSWORD -d "$RULE_PARAMS" $REST_API_URL/mgm
 
 ## Viewing Current Group Approval Rules
 
-To retrieve all applied group approval rules, use the [mgm/{holdingidentityshorthash}/approval/rules GET method](../../../rest-api/C5_OpenAPI.html#tag/MGM-API/operation/get_mgm__holdingidentityshorthash__approval_rules).
+To retrieve all applied group approval rules, use the [mgm/{holdingidentityshorthash}/approval/rules GET method](https://docs.r3.com/en/platform/corda/5.0/reference/rest-api/C5_OpenAPI.html#tag/MGM-API/operation/get_mgm__holdingidentityshorthash__approval_rules).
```
Comment (nadinequinr3) [inline content/en/platform/corda/5.0/application-networks/managing/registration-requests/configuring-manual-approval-rules.md:27]:
```suggestion
To retrieve all applied group approval rules, use the [mgm/{holdingidentityshorthash}/approval/rules GET method](../../../reference/rest-api/C5_OpenAPI.html#tag/MGM-API/operation/get_mgm__holdingidentityshorthash__approval_rules).
```

File: content/en/platform/corda/5.0/application-networks/managing/registration-requests/configuring-manual-approval-rules.md [hunk 2]
```diff
@@ -41,7 +41,7 @@ This method returns the rules in the following format:
 
 ## Deleting a Group Approval Rule
 
-To delete an applied group approval rule, pass the ID of the rule to the [mgm/{holdingidentityshorthash}/approval/rules/{ruleid} DELETE method](../../../rest-api/C5_OpenAPI.html#tag/MGM-API/operation/delete_mgm__holdingidentityshorthash__approval_rules__ruleid_). You can retrieve the ID of a rule from the response of creating the rule, or from the response of the GET method described in [Viewing Current Group Approval Rules]({{< relref "#viewing-current-group-approval-rules" >}}).
+To delete an applied group approval rule, pass the ID of the rule to the [mgm/{holdingidentityshorthash}/approval/rules/{ruleid} DELETE method](https://docs.r3.com/en/platform/corda/5.0/reference/rest-api/C5_OpenAPI.html#tag/MGM-API/operation/delete_mgm__holdingidentityshorthash__approval_rules__ruleid_). You can retrieve the ID of a rule from the response of creating the rule, or from the response of the GET method described in [Viewing Current Group Approval Rules]({{< relref "#viewing-current-group-approval-rules" >}}).
```
Comment (nadinequinr3) [inline content/en/platform/corda/5.0/application-networks/managing/registration-requests/configuring-manual-approval-rules.md:44]:
```suggestion
To delete an applied group approval rule, pass the ID of the rule to the [mgm/{holdingidentityshorthash}/approval/rules/{ruleid} DELETE method](../../../reference/rest-api/C5_OpenAPI.html#tag/MGM-API/operation/delete_mgm__holdingidentityshorthash__approval_rules__ruleid_). You can retrieve the ID of a rule from the response of creating the rule, or from the response of the GET method described in [Viewing Current Group Approval Rules]({{< relref "#viewing-current-group-approval-rules" >}}).
```

File: content/en/platform/corda/5.0/application-networks/managing/registration-requests/pre-auth/configuring-pre-auth-rules.md [hunk 3]
```diff
@@ -26,7 +26,7 @@ curl -k -u $REST_API_USER:$REST_API_PASSWORD -d $RULE_PARAMS $REST_API_URL/mgm/$
 
 ## Viewing Current Pre-Auth Approval Rules
 
-To retrieve all created pre-auth approval rules, use the [mgm/{holdingidentityshorthash}/approval/rules/preauth GET method](../../../rest-api/C5_OpenAPI.html#tag/MGM-API/operation/get_mgm__holdingidentityshorthash__approval_rules_preauth).
+To retrieve all created pre-auth approval rules, use the [mgm/{holdingidentityshorthash}/approval/rules/preauth GET method](https://docs.r3.com/en/platform/corda/5.0/reference/rest-api/C5_OpenAPI.html#tag/MGM-API/operation/get_mgm__holdingidentityshorthash__approval_rules_preauth).
```
Comment (nadinequinr3) [inline content/en/platform/corda/5.0/application-networks/managing/registration-requests/pre-auth/configuring-pre-auth-rules.md:29]:
```suggestion
To retrieve all created pre-auth approval rules, use the [mgm/{holdingidentityshorthash}/approval/rules/preauth GET method](../../../reference/rest-api/C5_OpenAPI.html#tag/MGM-API/operation/get_mgm__holdingidentityshorthash__approval_rules_preauth).
```

File: content/en/platform/corda/5.0/application-networks/managing/registration-requests/pre-auth/configuring-pre-auth-rules.md [hunk 4]
```diff
@@ -15,7 +15,7 @@ As described in [Registration Approval]({{< relref "../_index.md#pre-authenticat
 
 ## Adding a Pre-Auth Approval Rule
 
-To add an approval rule for registrations containing a valid pre-auth token, use the [mgm/{holdingidentityshorthash}/approval/rules/preauth POST method](../../../rest-api/C5_OpenAPI.html#tag/MGM-API/operation/post_mgm__holdingidentityshorthash__approval_rules_preauth) of the REST API.
+To add an approval rule for registrations containing a valid pre-auth token, use the [mgm/{holdingidentityshorthash}/approval/rules/preauth POST method](https://docs.r3.com/en/platform/corda/5.0/reference/rest-api/C5_OpenAPI.html#tag/MGM-API/operation/post_mgm__holdingidentityshorthash__approval_rules_preauth) of the REST API.
```
Comment (nadinequinr3) [inline content/en/platform/corda/5.0/application-networks/managing/registration-requests/pre-auth/configuring-pre-auth-rules.md:18]:
```suggestion
To add an approval rule for registrations containing a valid pre-auth token, use the [mgm/{holdingidentityshorthash}/approval/rules/preauth POST method](../../../rest-api/C5_OpenAPI.html#tag/MGM-API/operation/post_mgm__holdingidentityshorthash__approval_rules_preauth) of the REST API.
```

File: content/en/platform/corda/5.0/application-networks/managing/registration-requests/pre-auth/configuring-pre-auth-rules.md [hunk 5]
```diff
@@ -43,7 +43,7 @@ This method returns the rules in the following format:
 
 ## Deleting a Pre-Auth Approval Rule
 
-To delete a pre-auth approval rule, pass the ID of the rule to the [mgm/{holdingidentityshorthash}/approval/rules/preauth/{ruleid} DELETE method](../../../rest-api/C5_OpenAPI.html#tag/MGM-API/operation/delete_mgm__holdingidentityshorthash__approval_rules_preauth__ruleid_). You can retrieve the ID of a rule from the response of creating the rule, or from the response of the GET method described in [Viewing Current Pre-Auth Approval Rules]({{< relref "#viewing-current-pre-auth-approval-rules" >}}).
+To delete a pre-auth approval rule, pass the ID of the rule to the [mgm/{holdingidentityshorthash}/approval/rules/preauth/{ruleid} DELETE method](https://docs.r3.com/en/platform/corda/5.0/reference/rest-api/C5_OpenAPI.html#tag/MGM-API/operation/delete_mgm__holdingidentityshorthash__approval_rules_preauth__ruleid_). You can retrieve the ID of a rule from the response of creating the rule, or from the response of the GET method described in [Viewing Current Pre-Auth Approval Rules]({{< relref "#viewing-current-pre-auth-approval-rules" >}}).
```
Comment (nadinequinr3) [inline content/en/platform/corda/5.0/application-networks/managing/registration-requests/pre-auth/configuring-pre-auth-rules.md:46]:
```suggestion
To delete a pre-auth approval rule, pass the ID of the rule to the [mgm/{holdingidentityshorthash}/approval/rules/preauth/{ruleid} DELETE method](../../../rest-api/C5_OpenAPI.html#tag/MGM-API/operation/delete_mgm__holdingidentityshorthash__approval_rules_preauth__ruleid_). You can retrieve the ID of a rule from the response of creating the rule, or from the response of the GET method described in [Viewing Current Pre-Auth Approval Rules]({{< relref "#viewing-current-pre-auth-approval-rules" >}}).
```

File: content/en/platform/corda/5.0/deploying-operating/config-users/managing-roles.md [hunk 6]
```diff
@@ -34,7 +34,7 @@ For information about creating these roles maually, see the [Manual Bootstrappin
 
 ## Querying Permissions via REST
 
-To retrieve permissions matching certain query criteria, use the [get_permission](https://docs.r3.com/en/platform/corda/5.0/rest-api/C5_OpenAPI.html#tag/RBAC-Permission-API/operation/get_permission) API call.
+To retrieve permissions matching certain query criteria, use the [get_permission](https://docs.r3.com/en/platform/corda/5.0/reference/rest-api/C5_OpenAPI.html#tag/RBAC-Permission-API/operation/get_permission) API call.
```
Comment (nadinequinr3) [inline content/en/platform/corda/5.0/deploying-operating/config-users/managing-roles.md:37]:
```suggestion
To retrieve permissions matching certain query criteria, use the [get_permission](../../../reference/rest-api/C5_OpenAPI.html#tag/RBAC-Permission-API/operation/get_permission) API call.
```

File: content/en/platform/corda/5.0/application-networks/managing/registration-requests/reviewing-registration-requests.md [hunk 7]
```diff
@@ -58,7 +58,7 @@ You can only approve requests that are in the `PENDING_MANUAL_APPROVAL` status.
 ## Declining a Request
 
 To decline a request, pass the ID of the request and a reason to the [POST method of the
-mgm/{holdingidentityshorthash}/decline/{requestid} endpoint](../../../rest-api/C5_OpenAPI.html#tag/MGM-API/operation/post_mgm__holdingidentityshorthash__decline__requestid_). You can retrieve the ID of a request from the response of the GET endpoint described in [Viewing Requests Pending Manual Approval]({{< relref "#viewing-requests-pending-manual-approval" >}}). 
+mgm/{holdingidentityshorthash}/decline/{requestid} endpoint](https://docs.r3.com/en/platform/corda/5.0/reference/rest-api/C5_OpenAPI.html#tag/MGM-API/operation/post_mgm__holdingidentityshorthash__decline__requestid_). You can retrieve the ID of a request from the response of the GET endpoint described in [Viewing Requests Pending Manual Approval]({{< relref "#viewing-requests-pending-manual-approval" >}}).
```
Comment (nadinequinr3) [inline content/en/platform/corda/5.0/application-networks/managing/registration-requests/reviewing-registration-requests.md:61]:
```suggestion
mgm/{holdingidentityshorthash}/decline/{requestid} endpoint](../../../reference/rest-api/C5_OpenAPI.html#tag/MGM-API/operation/post_mgm__holdingidentityshorthash__decline__requestid_). You can retrieve the ID of a request from the response of the GET endpoint described in [Viewing Requests Pending Manual Approval]({{< relref "#viewing-requests-pending-manual-approval" >}}). 
```

File: content/en/platform/corda/5.0/application-networks/managing/registration-requests/reviewing-registration-requests.md [hunk 8]
```diff
@@ -46,7 +46,7 @@ curl -k -u $REST_API_USER:$REST_API_PASSWORD $REST_API_URL/mgm/$MGM_HOLDING_ID/a
 ## Approving a Request
 
 To approve a rquest, pass the ID of the request to the [POST method of the
-mgm/{holdingidentityshorthash}/approve/{requestid} endpoint](../../../rest-api/C5_OpenAPI.html#tag/MGM-API/operation/post_mgm__holdingidentityshorthash__approve__requestid_). You can retrieve the ID of a request from the response of the GET endpoint described in [Viewing Requests Pending Manual Approval]({{< relref "#viewing-requests-pending-manual-approval" >}}). 
+mgm/{holdingidentityshorthash}/approve/{requestid} endpoint](https://docs.r3.com/en/platform/corda/5.0/reference/rest-api/C5_OpenAPI.html#tag/MGM-API/operation/post_mgm__holdingidentityshorthash__approve__requestid_). You can retrieve the ID of a request from the response of the GET endpoint described in [Viewing Requests Pending Manual Approval]({{< relref "#viewing-requests-pending-manual-approval" >}}).
```
Comment (nadinequinr3) [inline content/en/platform/corda/5.0/application-networks/managing/registration-requests/reviewing-registration-requests.md:49]:
```suggestion
mgm/{holdingidentityshorthash}/approve/{requestid} endpoint](../../../reference/rest-api/C5_OpenAPI.html#tag/MGM-API/operation/post_mgm__holdingidentityshorthash__approve__requestid_). You can retrieve the ID of a request from the response of the GET endpoint described in [Viewing Requests Pending Manual Approval]({{< relref "#viewing-requests-pending-manual-approval" >}}). 
```

File: content/en/platform/corda/5.0/developing-applications/packaging/code-signing/_index.md [hunk 9]
```diff
@@ -14,7 +14,7 @@ section_menu: corda5
 
 CorDapps are packaged as CPKs, CPBs, and CPIs. Each of these packages must be signed with a Code Signing certificate.
 The signatures are then verified when a CPI is installed. They are verified against the certificates
-uploaded to the [Corda REST API Docs](https://docs.r3.com/en/platform/corda/5.0/rest-api/C5_OpenAPI.html#tag/Certificates-API) API.
+uploaded to the [Corda REST API Docs](https://docs.r3.com/en/platform/corda/5.0/reference/rest-api/C5_OpenAPI.html#tag/Certificates-API) API.
```
Comment (nadinequinr3) [inline content/en/platform/corda/5.0/developing-applications/packaging/code-signing/_index.md:17]:
```suggestion
uploaded to the [Corda REST API Docs](../../../reference/rest-api/C5_OpenAPI.html#tag/Certificates-API) API.
```

File: content/en/platform/corda/5.0/developing-applications/packaging/code-signing/custom-cert.md [hunk 10]
```diff
@@ -19,7 +19,7 @@ However, you can configure the plugin to use a custom certificate using the `cor
 
 {{< note >}}
 When a custom certificate is used, it will need to be uploaded to the cluster using the
-[/api/v1/certificates/cluster/code-signer API endpoint](https://docs.r3.com/en/platform/corda/5.0/rest-api/C5_OpenAPI.html#tag/Certificates-API/operation/put_certificates_cluster__usage_).
+[/api/v1/certificates/cluster/code-signer API endpoint](https://docs.r3.com/en/platform/corda/5.0/reference/rest-api/C5_OpenAPI.html#tag/Certificates-API/operation/put_certificates_cluster__usage_).
```
Comment (nadinequinr3) [inline content/en/platform/corda/5.0/developing-applications/packaging/code-signing/custom-cert.md:22]:
```suggestion
[/api/v1/certificates/cluster/code-signer API endpoint](../../../reference/rest-api/C5_OpenAPI.html#tag/Certificates-API/operation/put_certificates_cluster__usage_).
```

File: content/en/platform/corda/5.0/application-networks/managing/registration-requests/pre-auth/configuring-pre-auth-rules.md (hunk 5 above)
Comment (amiegrace12) [reply content/en/platform/corda/5.0/application-networks/managing/registration-requests/pre-auth/configuring-pre-auth-rules.md:46]: Do we need reference here?

**Review by colin-hillis — APPROVED** [review]
LGTM 0 you will need Nadine to sign off on this too, as she suggested a change. Thanks Amie

**Review by nadinequinr3 — APPROVED** [review]

**Pull request merged**